### PR TITLE
Updating deps

### PR DESCRIPTION
**Pull request by mdfleury-wbd**
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Documentation change

## Checklist

- [ ] My code follows conforms to the coding standards.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

**Comment by mdfleury-wbd** [comment]
Updated the package-lock to v3, which drops support for node14 and is a lot smaller.

File: ui/package.json [hunk 1]
```diff
@@ -20,7 +20,7 @@
     "@lingui/react": "^4.13.0",
     "@mui/icons-material": "^5.16.7",
     "@mui/lab": "5.0.0-alpha.132",
-    "@mui/material": "^5.13.3",
+    "@mui/material": "5.14.11",
```
Comment (g-marconet) [inline ui/package.json:23]: What's the reason for this change?

Comment (mdfleury-wbd) [reply ui/package.json:23]: That's the highest version we can use before the breaking change, so I pinned it at that.
https://github.com/wmcso/artemis/wiki/Updating-Dependencies/#held-back-updates

**Review by g-marconet — APPROVED** [review]
LGTM
